### PR TITLE
Update turn-on-or-off-offline-messages-for-admins.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-offline-messages-for-admins.md [hunk 1]
```diff
@@ -37,40 +37,19 @@ For more information, see [Use offline messaging in Skype for Business](https://
...
-   > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
-
-  ```PowerShell
-  Import-Module -Name MicrosoftTeams
-  $credential = Get-Credential
-  $session = New-CsOnlineSession -Credential $credential
-  Import-PSSession $session
-  ```
-
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-offline-messages-for-admins.md:40]:
```suggestion
> [!NOTE]
```

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-offline-messages-for-admins.md [hunk 2]
```diff
@@ -37,40 +37,19 @@ For more information, see [Use offline messaging in Skype for Business](https://
...
+  > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-offline-messages-for-admins.md:41]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**